packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.js

```diff
@@ -75,25 +77,30 @@ class OverridesResolverPlugin {
         }
     }
 
-    toOverrideRelative(_path) {
-        const override = this.findOverride(_path)
-        return _path.substring(override.length + 1)
+    toOverrideRelative(filepath) {
+        const override = this.findOverride(filepath)
+        return filepath.substring(override.length + 1)
     }
 
-    findOverride(_path) {
+    findOverride(filepath) {
         return this._allSearchDirs.find((override) => {
-            return _path.indexOf(override) === 0
+            return filepath.indexOf(override) === 0
         })
     }
 
-    isFromExtends(request, _path) {
+    isFromExtends(request, filepath) {
         // in npm namespaces like `@salesforce/<pkg>` we need to ignore the first slash
-        const basePkgIndex = request?.startsWith('@') ? 1 : 0
+        var [z, o] = request.split('/')
```

Comment:
I have a strong objection to single-letter variables, because as code evolves there is a significant risk that `o` and `z` here get used more, and as that happens, refactoring and searching for existing references becomes exponentially harder when they are single characters that can exist many times in other unrelated places in the code